e2e_tests/account_test.py

```diff
@@ -145,17 +145,12 @@ def test_update_account():
 
 
 def test_update_credit_account():
-    account_id = create_credit_account_for_business().data.id
+    account_id = "792799"
```

Comment:
Having hardcoded dynamic info in tests would make them irreproducible in certain cases. Do we need this to be hardcoded? If we do this probably needs to be an env variable